doc/api/number/number-formatter.md

```diff
@@ -122,6 +122,28 @@ formatter( 0.00012345 );
 // > "0.000123"
 ```
 
+Format percentages:
+
+```javascript
+var enFormatter = Globalize( "en" ).numberFormatter({
+  style: "percent",
+  minimumFractionDigits: 2,
+  maximumFractionDigits: 2
+});
+
+var frFormatter = Globalize( "fr" ).numberFormatter({
+  style: "percent",
+  minimumFractionDigits: 2,
+  maximumFractionDigits: 2
```

Comment:
In this example, since the locale only affects the decimal separator, I'd use different values for the fraction digits here, to demonstrate a bit more variation.